static/app/components/commandPalette/types.tsx

```diff
@@ -0,0 +1,28 @@
+import type {ReactNode} from 'react';
+import type {LocationDescriptor} from 'history';
+
+export type CommandPaletteAction = {
+  /** Unique identifier for this action */
+  key: string;
+  /** Primary text shown to the user */
+  label: string;
+  /** Icon to render for this action */
+  actionIcon?: ReactNode;
```

Comment:
It's already an icon :)
```suggestion
  icon?: ReactNode;
```